lib/web/controllers/project_controller.ex

```diff
@@ -331,9 +331,13 @@ defmodule BorsNG.ProjectController do
             {:ok, "Successfully added #{user.login} as a reviewer"}
         end
     end
-    conn
-    |> put_flash(state, msg)
-    |> redirect(to: project_path(conn, :settings, project))
+    case conn do
+      nil -> msg
```

Comment:
Why is `conn` winding up as `nil` here? This seems like a sign that there's another bug outside of this code.

Reply:
Oops, thanks for catching this. That's a change I made for easier development that I forgot to undo. Just pushed a change to put this back the way it was.

For context, though not needed for this PR, I used a "friendlier" version of ServerMock (in conjunction with the REPL in `iex -S mix phx.server`) to get a better idea of how Bors works and to do development. I had trouble getting the nested maps on ServerMock in the right state otherwise. https://github.com/bors-ng/bors-ng/pull/785/commits/babca4b78908b058451a554a08877d096a3c10c8

Reply:
That's pretty cool. If you still want it, you can open another PR for it. I'd be happy to accept your `FriendlyMock` system into bors for developer convenience.

Reply:
Cool, will do!